### PR TITLE
User managment

### PR DESCRIPTION
**Pull request by MartinHinz**

### CODE REVIEW

**Comment by MartinHinz** [comment]
Added basic user managment and tests. Some integration tests using javascript had not worked due to problems with selenium, so I removed them again.

File: app/controllers/user_profiles_controller.rb [hunk 1]
```diff
@@ -62,11 +76,26 @@ def destroy
...
     end
 
     # Only allow a list of trusted parameters through.
     def user_profile_params
-      params.require(:user_profile).permit(:first_name, :last_name, :user_id)
+      params.fetch(:user_profile, {}).permit(
+      :first_name,
+      :last_name,
+      :user_id,
+      {user_attributes: [
```
Comment (joeroe) [inline app/controllers/user_profiles_controller.rb:92]: Need to add `admin` here.

**Review by joeroe — CHANGES_REQUESTED** [review]
For me there were several errors preventing me from submitting the 'new user profile' form:

* user_profile_params is missing `admin`
* "User passphrase is wrong"
* "User can't be blank"

I don't know if you also want to address #266, #267 and (new) #281 before merging?

File: app/controllers/user_profiles_controller.rb (hunk 1 above)
Comment (MartinHinz) [reply app/controllers/user_profiles_controller.rb:92]: Thanks, added.

**Comment by joeroe** [comment]
@MartinHinz Sorry, I meant #277 and #276.

**Comment by MartinHinz** [comment]
Thanks, now it makes much more sense. Please put this PR aside (I will not like to close it, since the discussion here is very helpful). I will implement your suggestions and come back when finished.

**Comment by joeroe** [comment]
The above addresses the errors when submitting user profile forms, as well as #277 and #281.

Remaining/new issues relating to user management: #276, #315, #316

**Pull request merged**